homeassistant/components/analytics/manifest.json

```diff
@@ -4,6 +4,7 @@
   "documentation": "https://www.home-assistant.io/integrations/analytics",
   "codeowners": ["@home-assistant/core", "@ludeeus"],
   "dependencies": ["api", "websocket_api"],
+  "requirements": ["sqlalchemy==1.4.41"],
```

Comment:
Why is this needed?

Reply:
Without it `hass -c config` gives:
```
2022-09-11 18:16:39.040 ERROR (MainThread) [homeassistant.setup] Setup failed for analytics: Unable to import component: No module named 'sqlalchemy'
2022-09-11 18:16:40.105 INFO (MainThread) [homeassistant.setup] Setting up image
2022-09-11 18:16:40.107 INFO (MainThread) [homeassistant.setup] Setup of domain image took 0.0 seconds
2022-09-11 18:16:40.108 INFO (MainThread) [homeassistant.setup] Setting up person
2022-09-11 18:16:40.110 INFO (MainThread) [homeassistant.setup] Setup of domain person took 0.0 seconds
2022-09-11 18:16:40.110 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of onboarding. Setup failed for dependencies: analytics
2022-09-11 18:16:40.111 ERROR (MainThread) [homeassistant.setup] Setup failed for onboarding: (DependencyError(...), 'Could not setup dependencies: analytics')
2022-09-11 18:16:40.111 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of frontend. Setup failed for dependencies: onboarding
2022-09-11 18:16:40.111 ERROR (MainThread) [homeassistant.setup] Setup failed for frontend: (DependencyError(...), 'Could not setup dependencies: onboarding')
```

Reply:
Sure, but why?
This "fix" seems wrong.

Reply:
@ludeeus is right, this is not the right fix for the problem. @d-walsh Can you open an issue where you explain how to reproduce the import problem?